.gitlab-ci.yml

```diff
@@ -0,0 +1,332 @@
+  rules:
+    - when: manual
+
+deploy-to-mig:
+  image: mcr.microsoft.com/azure-cli
+  stage: migration
+  needs: ["build-and-push-images"]
+  environment:
+    name: migration
+  before_script:
+    - az extension add --name containerapp
+    - az login --service-principal -u${ARM_CLIENT_ID} -p${ARM_CLIENT_SECRET} -t${ARM_TENANT_ID}
+  script:
+    - |
+      if [[ "$CI_COMMIT_BRANCH" == "$CI_DEFAULT_BRANCH" ]]; then
+        tag=""
+        echo "Running on default branch '$CI_DEFAULT_BRANCH': tag = 'latest'"
+      else
+        tag=":$CI_COMMIT_REF_SLUG"
+        echo "Running on branch '$CI_COMMIT_BRANCH': tag = $tag"
+      fi
+    - echo "Creating new revision of Container App"
+    - |
+      cat > revision.yml <<EOF
+      properties:
+        template:
+          revisionSuffix: ${CI_PIPELINE_ID}
+          containers:
+            - image: acreccuksdev.azurecr.io/microsites-nginx-drupal$tag
+              name: nginx
+              resources:
+                cpu: 0.25
+                memory: 0.5Gi
+              volumeMounts:
+              - mountPath: /drupal/web/sites/default/files
+                volumeName: filesharevol
+              env:
+              - name: X_ROBOTS_TAG
+                value: noindex
+            - image: acreccuksdev.azurecr.io/microsites-drupal-fpm$tag
+              name: drupal
+              resources:
+                cpu: 0.75
+                memory: 1.5Gi
+              volumeMounts:
+              - mountPath: /drupal/web/sites/default/files
+                volumeName: filesharevol
+              - mountPath: /drupal/data/default/private
+                volumeName: privsharevol
+              env:
+              - name: MYSQL_HOST
+                value: mariadb-ecc-uks-mig.mariadb.database.azure.com
+              - name: MYSQL_USER
+                value: mariadb-root
+              - name: MYSQL_DATABASE
+                value: drupal_microsites
+              - name: MYSQL_PASSWORD
+                secretRef: mysql-password
+              - name: OPENID_CONNECT_PARAMS
+                secretRef: openid-connect-params
+      EOF
+    - az containerapp revision copy --name microsites --resource-group rg-ecc-microsites-uks-mig --yaml revision.yml --subscription "Essex County Council (Microsites)"
+  rules:
+    - when: manual
+
+deploy-to-preprod:
+  image: mcr.microsoft.com/azure-cli
+  stage: preproduction
+  needs: ["deploy-to-dev"]
+  environment:
+    name: preproduction
+    url: https://essex-gov-pp.nomensa.xyz/
+  before_script:
+    - az extension add --name containerapp
+    - az login --service-principal -u${ARM_CLIENT_ID} -p${ARM_CLIENT_SECRET} -t${ARM_TENANT_ID}
+  script:
+    - echo "Importing image from Development  into Pre-Production"
+    - az acr import --force --name acreccukspre --source acreccuksdev.azurecr.io/microsites-nginx-drupal:${CI_COMMIT_REF_SLUG} --image microsites-nginx-drupal:${CI_COMMIT_REF_SLUG} --subscription "Essex County Council (Common)"
```

Comment:
@MatthewWilkes should the subscription be `--subscription "Essex County Council (Microsites)"` here? I was copied and pasted from .gov's gitlab-ci file.

Reply:
No, this is referencing the container registries, which are common across all three sites.